### DISCUSSION TITLE
setSelectionFilter失效

### DISCUSSION THREAD

**Discussion by handsomeCarl** [antvis/X6#3037] [Q&A]
setSelectionFilter 设置筛选的节点，再次选择的时候依然能选中，这个有人遇到过吗

**Comment by NewByVector** [comment]
X6 哪个版本呢？可以整理一个可以复现的 codesanbox，提个 issue。

**Reply by handsomeCarl** [reply]
"@antv/x6": "^1.31.4",    我看看整理个codesanbox
